### PR TITLE
Add a few more wildcard redirects

### PR DESCRIPTION
**Pull request by ethax-ross**
These appear to have been the most frequent 404s recently.

### CODE REVIEW

**Review by leoapost — APPROVED** [review]

File: config/redirects.yml [hunk 1]
```diff
@@ -287,7 +287,11 @@ redirects:
   "/life-as-a-teacher/real-life-experiences/(*story)": "/my-story-into-teaching"
   "/funding-and-salary(*section)": "/funding-your-training"
   "/regional-make-a-difference-in-(*area)": "/"
-
+  "/how-to-apply(*method)": "/steps-to-become-a-teacher"
```
Comment (MylesJarvis) [inline config/redirects.yml:290]: /tips-on-applying-for-teacher-training might be a better one here

Comment (ethax-ross) [reply config/redirects.yml:290]: Same as well for this one? `/premier-plus-adviser(*tip)`

Comment (MylesJarvis) [reply config/redirects.yml:290]: Think that one might be best suited as /three-things-to-help-you-get-into-teaching. I just used wayback machine to check its origins, it's solely focussed on TTAs

Comment (peteryates) [reply config/redirects.yml]: Sounds like someone who [advises on hotel deals at Premier Inn](https://www.premierinn.com/gb/en/sleep/premier-plus.html) 😅

**Review by peteryates — APPROVED** [review]

**Pull request merged**